Submodels/MTSOL_call.py

```
@@ -1,20 +1,613 @@
+
+        # Initialize exit flag
+        exit_flag = ExitFlag.NON_CONVERGENCE.value
+
+        # Keep converging until the iteration count exceeds the limit
+        while iter_counter < self.ITER_LIMIT:
+            # Check the exit flag to see if the solution has converged
+            # If the solution has converged, break out of the iteration loop
+            exit_flag = self.WaitForCompletion(type=1)
+            if exit_flag == ExitFlag.SUCCESS.value:
+                return exit_flag, iter_counter
+            else:
```

Ruff (0.8.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)